nilearn/tests/test_testing.py

```diff
@@ -1,10 +1,36 @@
 import itertools
-
 import numpy as np
 
 from nose.tools import assert_equal, assert_raises
 
-from nilearn._utils.testing import generate_fake_fmri
+from nilearn._utils.testing import generate_fake_fmri, with_memory_profiler
+from nilearn._utils.testing import assert_memory_less_than, assert_raises_regex
+
+
+def create_object(size):
+    """Just create and return an object containing `size` bytes."""
+    mem_use = b'a' * size
```

Comment:
This generates a flake8 issue on travis (non used variable). Replacing by `b'a' * size` makes codacy fails...